FXyz-Core/src/main/java/org/fxyz3d/shapes/polygon/SkinningMesh.java

```diff
@@ -203,4 +207,38 @@ public void update() {
 
         jointsTransformDirty = false;
     }
+
+    // showSkeleton
+    private final BooleanProperty showSkeleton = new SimpleBooleanProperty(this, "showSkeleton") {
+        @Override
+        protected void invalidated() {
+            if (joints == null) {
+                return;
+            }
+            if (get()) {
+                joints.forEach(SkinningMesh.this::addJointChain);
+            } else {
+                joints.forEach(SkinningMesh.this::removeJointChain);
+            }
+        }
+    };
+    public final BooleanProperty showSkeletonProperty() {
+       return showSkeleton;
+    }
+    public final boolean isShowSkeleton() {
+       return showSkeleton.get();
+    }
+    public final void setShowSkeleton(boolean value) {
+        showSkeleton.set(value);
+    }
+
+    private void addJointChain(Joint joint) {
+        joint.getChildren().removeIf(JointChain.class::isInstance);
```

Comment:
Any reason not to use `removeJointChain` here to make it easy to see that adding a chain first removes the current one?